presto-main/src/main/java/com/facebook/presto/operator/Driver.java

```diff
@@ -234,20 +237,17 @@ private void processNewSources()
         long maxRuntime = duration.roundTo(TimeUnit.NANOSECONDS);
 
         Optional<ListenableFuture<?>> result = tryWithLock(100, TimeUnit.MILLISECONDS, () -> {
-            // if the driver is blocked we don't need to continue processing
```

Comment:
Now that we register listeners once we don't need these checks. We needed it previously because we wanted to make sure that when we set the new listeners we would be done by the previous listeners we set.